en/ui-system/components/editor/widget.md

```diff
@@ -76,6 +78,29 @@ To make sure you can update node's position or size during runtime:
 1. Set __Align Mode__ property of Widget to `ONCE`, so it will only align during onEnable process.
 2. Use Widget's API to update node's position and size, for example updating Widget's `top`, `bottom`, `left`, `right` instead of node's `x`, `y`, `width`, `height`.
 
+### Modify the alignment distance through script code
```

Comment:
Modify the alignment distance through script code
-->
Modify the alignment distance through in code